easycv/models/backbones/yolo6_blocks.py

```diff
@@ -0,0 +1,321 @@
+#!/usr/bin/env python3
```

Comment:
merge and reuse repvgg_yolox_backbone and yolo6_blocks，reuse op move to model/utils.py

Reply:
done